specification/sql/resource-manager/readme.md

```diff
@@ -127,6 +126,7 @@ input-file:
 - Microsoft.Sql/preview/2020-11-01-preview/Databases_legacy.json 
 - Microsoft.Sql/preview/2020-11-01-preview/LongTermRetentionBackups.json
 - Microsoft.Sql/preview/2020-11-01-preview/LongTermRetentionPolicies.json
+- Microsoft.Sql/preview/2020-11-01-preview/PrivateEndpointConnections.json
```

Comment:
not sure what validation error will be fixed.  is there any different between 2 privatelintendpointconnection?

Reply:
without my PR , the validation errors are:
https://github.com/Azure/azure-rest-api-specs/pull/13321/checks?check_run_id=2060056651

I used the 2020-11-01 private connection spec to replace the 2018-06-01 one. I look through it and did not find any API contract change. The only change is private connection state is switched from string to enum string.

Reply:
the spec files differences are: https://www.diffchecker.com/bFMuIpaE
From my perspective, they are the same.